### PR TITLE
Add ownership by team to applications section

### PR DESCRIPTION
**Pull request by whoojemaflip**
First draft of a section to look at applications sorted by owning team.

[Trello](https://trello.com/c/bM1an40m/48-make-it-easy-to-determine-ownership-of-a-repo-project)

### CODE REVIEW

File: source/apps/by_team.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (tijmenb) [inline source/apps/by_team.html.md.erb]: Can we use dashes in the URL?

File: source/apps/by_team.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+layout: application_layout
+title: Application Ownership by Team
```
Comment (tijmenb) [inline source/apps/by_team.html.md.erb]: I think the other pages use non-titleised headings, maybe `Application ownership by team` is more consistent?

File: config.rb [hunk 3]
```diff
@@ -50,6 +50,10 @@ def opsmanual_pages
       .group_by { |page| page.data.section }
   end
 
+  def applications_by_team
+    active_app_pages.group_by(&:team)
```
Comment (tijmenb) [inline config.rb]: If we make this return an object for the collection, we can move some of the logic in the views to there.

**Comment by whoojemaflip** [comment]
@tijmenb edits made.

**Pull request merged**